src/components/GameController.tsx

```diff
@@ -55,10 +54,10 @@ export default class GameController extends PureComponent<Props> {
 
     switch (applicationState && applicationState.constructor) {
       case playerA.ReadyToSendPreFundSetupA:
-        return <ProposeGamePage message="ready to propose game" />;
+        return <GameProposedPage message="Waiting for opponent to accept game" />;
 
       case playerA.WaitForPreFundSetupB:
-        return <ProposalSentPage message="Waiting for opponent to accept game" />;
+        return <GameProposedPage message="Waiting for opponent to accept game" />;
```

Comment:
This no longer uses a message prop, so you should delete this prop (both in `GameProposedPage` and here)

Reply:
I think we probably will probably use this `message` in the longer term. The idea is for what's currently the `message` to be displayed in the status bar at the bottom of the page.

Reply:
Gotcha - I think the status bar will likely be something that is rendered by the GameController, not child components, but until we figure that structure out, I can leave it as a comment